lrpar/src/lib/ctbuilder.rs

```diff
@@ -92,6 +92,59 @@ where
 {
 }
 
+/// The scope of public visibility.
+#[derive(Clone,PartialEq,Eq,Debug,Hash)]
+pub enum Scope {
+    /// Represents `pub(crate)`
+    Crate,
+    /// Represents `pub(self)`
+    _Self,
```

Comment:
A `_` prefix means something in Rust ("unused"), so `Self_` is probably preferable.